plugins/ipam/host-local/backend/allocator/config.go

```diff
@@ -48,7 +50,7 @@ type IPAMConfig struct {
 	DataDir    string         `json:"dataDir"`
 	ResolvConf string         `json:"resolvConf"`
 	Ranges     []RangeSet     `json:"ranges"`
-	IPArgs     []net.IP       `json:"-"` // Requested IPs from CNI_ARGS and args
+	IPArgs     []net.IP       `json:"-"` // Requested IPs from CNI_ARGS, args and capabilities
```

Comment:
It looks like the ips convention describes itself as being a list of CIDRs (don't like the name!), which can't go in net.IPs.  It might be that the convention is written wrongly, but something needs to change.

Reply:
Good question! Thanks @matthewdupre , I'll check this later ~

Reply:
@matthewdupre  Sorry for the late reply, I checked CNI convention and found that, for CNI_ARGS and args in network config, "ips" follow the spec as
```
Spec:
"ips": ["<ip>[/<prefix>]", ...]
Examples:
"ips": ["10.2.2.42/24", "2001:db8::5"]
The plugin may require the IP address to include a prefix length.
```
So I think ips can be filled with both IP and CIDR format.
But the spec of ips capability in runtime config only show a CIDR format, so maybe we should update the spec after merging this.

Reply:
@matthewdupre conventions say prefix is optional; but I really think host-local should accept prefix as well (and validate against ranges) for consistency with the static plugin.

Reply:
@matthewdupre Waiting for your reply, then I'll update this PR.

Reply:
Yeah, I agree with @dcbw; host-local should accept a host+prefix (and validate that it matches an existing range), so that it matches static-ip.

Reply:
OK, I will do this soon, thanks for all your suggestions.

Reply:
I read about the related lines in `static` plugin, it seems that `static` plugin only support CIDR format (ip with prefix) as the line [https://github.com/containernetworking/plugins/blob/655116585396cc26ca973db2d5daef66d821ad90/plugins/ipam/static/main.go#L216](https://github.com/containernetworking/plugins/blob/655116585396cc26ca973db2d5daef66d821ad90/plugins/ipam/static/main.go#L216), so I'm wondering whether we should support both IP and CIDR format in `host-local`? @squeed @dcbw 

The `ips` in `static` is more like a whole **IP config** including ip, subnet and gateway, but `ips` in `host-local` is just an **IP** because the other configs (subnet and gateway) can come from ranges.
So I think maybe the two `ips` are different, should we separate them in convention?

Reply:
conclusion: support both IP and CIDR format in `host-local` plugin

Reply:
I think it's probably okay to use the same convention for both. You are right, it's slightly awkward. But I think the higher-level intent is "I want this address please" which is the same for both.